contracts/libraries/GnosisAuction.sol

```diff
@@ -74,8 +74,10 @@ library GnosisAuction {
                 oTokenSellAmount.mul(10**10),
                 auctionDetails
                     .oTokenPremium
-            )
-                .div(10**(uint256(18).sub(auctionDetails.assetDecimals)));
+            );
+
+        // uint256 baseDecimals = auctionDetails.assetDecimals > 18 ? auctionDetails.assetDecimals : 18;
+        minBidAmount = minBidAmount.div(10**(uint256(24).sub(auctionDetails.assetDecimals)));
```

Comment:
shouldn't this scale the minBidAmount to match `assetDecimals`?
```
uint256 minBidAmount =
    DSMath.wmul(
        oTokenSellAmount.mul(10**10),
        auctionDetails.oTokenPremium
    );

minBidAmount = auctionDetails.assetDecimals > 18
    ? minBidAmount.mul(10**(auctionDetails.assetDecimals.sub(18)))
    : minBidAmount.div(
        10**(uint256(18).sub(auctionDetails.assetDecimals))
    );
```

Reply:
Thanks @0xTomoyo, this was a massive help and saved me time.
I will be fixing this up in https://github.com/ribbon-finance/ribbon-v2/pull/205 and closing out this PR.